source/features/cross-deleted-pr-branches.tsx

```diff
@@ -18,9 +18,9 @@ function init(): void | false {
 
 	const deletedBranchName = lastBranchAction.textContent!.trim();
 
-	const headReferenceLink = select('.head-ref a')!;
-	const repoRootUrl = headReferenceLink.href.split('/', 5).join('/');
-	const repoIsDeleted = headReferenceLink.textContent === 'unknown repository';
+	const headReference = select('.head-ref')!;
+	const repoIsDeleted = headReference.textContent === 'unknown repository';
+	const repoRootUrl = select('a', headReference)?.href.split('/', 5).join('/');
```

Comment:
```suggestion
	const repoRootUrl = select('a', headReference)?.pathname.split('/', 3).join('/');
```

Also I think if `repoRootUrl === undefined` then `repoIsDeleted === true`. In short, we don't need `repoIsDeleted` nor its exact check. Right?

Reply:
I am not sure I think github changed how they handle delete repos.

I will make a GHE comment and remove it when we drop refresh.

Reply:
No need to keep dead code around when we’re in doubt. We’re not fully sure this works on GHE either

Reply:
I am going to send this in a separate PR. This is a real fix.